angularjs-portal-home/src/main/webapp/partials/home-toggle.html

```diff
@@ -1,4 +1,23 @@
 <span class="layout-chooser" ng-if="toggle">
-  <li ng-class="{active : modeIs('list')}"><a  href="javascript:;" ng-click="switchMode('list')"><i class="fa fa-th"></i></a></li>
-  <li ng-class="{active : modeIs('widgets')}" ><a href="javascript:;" ng-click="switchMode('widgets')"><i class="fa fa-th-large"></i></a></li>
-</span>
\ No newline at end of file
+  <li ng-class="{active : modeIs('list')}">
+    <a  href="javascript:;" ng-click="switchMode('list')"
```

Comment:
I suspect that we ought to be doing something to this `<a` tag to improve its accessibility characteristics.  As in, a `title` or `ARIA`-something-or-other or a screen-reader-only alternative content for the tag?  Link text only visible to screen readers might be brilliant here, in that if I can't see the `fa-th`, what I really want is the text equivalent.  Maybe that should be happening on the `<i` tag?